### ISSUE TITLE
libssh minimum version?

### ISSUE THREAD

**Issue by mwinter-osr** [rtrlib/rtrlib#143]
The doc on the wiki says 0.5.0 is the minimum version of libssh.
The debian package (file debian/control) says 0.6.0 is the minimum

Which one is correct?
(the RPM PR was copied after debian package and currently checks for 0.6.0 as minimum as well)

(0.6.0 exclude older distros such as CentOS 6 and Ubuntu 12.04)

**Comment by waehlisch** [comment]
We had a discussion on this here #74.
We allow for libssh 0.5.0 (see CMakeLists.txt, which falls back to 0.5.0 if 0.6.0 is not available).
I suppose we configured 0.6.0 in rpm because we prefer more recent versions compared to outdated versions. @smlng?

**Comment by smlng** [comment]
The rtrlib was implemented against libssh 0.6.0, first. As we only use a certain part of the libssh API and there was some need to support rtrlib on Ubuntu 12.04 LTS (which had/has libssh 0.5.x only too), we found that its quite easy to lower the requirement on libssh version. 

However, that was after the initial Debian support was introduced, so it seems that we simply missed to adapt the Debian package description and dependencies that time. 

TL;DR: It should be possible to lower the requirement for libssh in both deb and rpm to version > 0.5.0 - note: its not possible to use a rtrlib that was build against libssh 0.5.x with libssh 0.6.y even though the requirement of > 0.5.0 is met -> there was an API change between 0.5 and 0.6.

**Comment by waehlisch** [comment]
@mroethke or @smlng can you fix and test? Thanks.

**Comment by randyqx** [comment]
well, here i am many years later, and on debian12, bookworm.  and get the dreaded
```
libssh >= 0.5.0 not found
```
while, but the libssh world seems to have moved forward a bit
```
r42# apt info libssh*
Package: libssh2-1
Version: 1.10.0-3+b1
```
clue bat solicited.  thanks.
